--- a/drivers/gpu/drm/panel/Kconfig
+++ b/drivers/gpu/drm/panel/Kconfig
@@ -105,6 +105,26 @@
 	  Say Y here if you want to support for BOE TV101WUM-LL2
 	  WUXGA PANEL DSI Video Mode panel

+config DRM_PANEL_CWU50_CM4
+	tristate "CWU50 panel for CM4"
+	depends on OF && DRM_MIPI_DSI && BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for the CWU50 panel
+	  specifically when used with the Compute Module 4.
+	  The panel has a 720x1280 resolution and uses 24-bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwu50-cm4.
+
+config DRM_PANEL_CWU50_CM5
+	tristate "CWU50 panel for CM5"
+	depends on OF && DRM_MIPI_DSI && BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for the CWU50 panel
+	  specifically when used with the Compute Module 5.
+	  The panel has a 720x1280 resolution and uses 24-bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwu50-cm5.
+
 config DRM_PANEL_EBBG_FT8719
 	tristate "EBBG FT8719 panel driver"
 	depends on OF
