--- /drivers/gpu/drm/panel/Kconfig	Fri Aug  1 08:41:27 2025
+++ /drivers/gpu/drm/panel/Kconfig	Fri Aug  1 09:23:13 2025
@@ -105,6 +96,50 @@
 	  Say Y here if you want to support for BOE TV101WUM-LL2
 	  WUXGA PANEL DSI Video Mode panel
 
+config DRM_PANEL_CWD686
+	tristate "CWD686 panel"
+ 	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for CWD686 panel.
+	  The panel has a 480x1280 resolution and uses 24 bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwd686.
+
+config DRM_PANEL_CWD686_CM3
+	tristate "CWD686_CM3 panel"
+ 	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for CWD686_CM3 panel.
+	  The panel has a 480x1280 resolution and uses 24 bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwd686-cm3.
+
+config DRM_PANEL_CWU50
+	tristate "CWU50 panel"
+ 	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for CWU50 panel.
+	  The panel has a 720x1280 resolution and uses 24 bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwu50.
+
+config DRM_PANEL_CWU50_CM3
+	tristate "CWU50_CM3 panel"
+ 	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for CWU50_CM3 panel.
+	  The panel has a 720x1280 resolution and uses 24 bit RGB per pixel.
+	  To compile this driver as a module, choose M here: the module
+	  will be called panel-cwu50-cm3.
+
 config DRM_PANEL_EBBG_FT8719
 	tristate "EBBG FT8719 panel driver"
 	depends on OF